config/custom-environment-variables.yaml

```diff
@@ -151,3 +151,5 @@ ecosystem:
     store: ECOSYSTEM_STORE
     # Badge service (needs to add a status and color)
     badges: ECOSYSTEM_BADGES
+    # Default registry to pull build containers from
+    # dockerRegistry: ECOSYSTEM_DOCKER_REGISTRY
```

Comment:
Don't you need to uncomment this line in order for user to configure via environment variable?